blaze/interactive.py

```diff
@@ -320,8 +320,21 @@ def table_length(expr):
     except ValueError:
         return int(expr.count())
 
+_warning_msg = """
+In version 0.11, Blaze's expresssion repr will return a standard 
+representation and will no longer implicitly compute.  Use the `peek()`
+method to see a preview of the expression's results.\
+"""
 
-Expr.__repr__ = expr_repr
+def _warning_repr(self):
+    # DeprecationWarnings are ignored by default, so we use a UserWarning here
+    # instead to ensure users are aware of this significant API change.
+    warnings.warn(_warning_msg)
+    return expr_repr(self)
+
+
+Expr.__repr__ = _warning_repr
+Expr.peek = lambda x: print(expr_repr(x))
```

Comment:
One comment I have is that if we are going to call `print`, could we accept a file argument so that users can pass a `StringIO` or other output stream?